src/Drivens/Dotlanche.Produto.Data/DatabaseContext/ProdutoDbContext.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Dotlanche.Produto.Data.DatabaseContext;
 
-internal class ProdutoDbContext : DbContext
+public class ProdutoDbContext : DbContext
```

Comment:
Isso pode afetar as confs do teste unitario quando for mergeado?

Reply:
Fiz o merge local dos testes de unidade e está tudo passando mesmo com as mudanças desse PR.
![image](https://github.com/user-attachments/assets/f0a112b1-b72a-49f9-afac-81239fedc915)